examples/train_val_test_split_example.ipynb

```diff
@@ -0,0 +1,195 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "9feb8e56",
+   "metadata": {},
+   "source": [
+    "# Using `train_val_test_split` for Rigorous Modeling\n",
+    "Thanks to [this](https://towardsdatascience.com/automatic-hyperparameter-tuning-with-sklearn-gridsearchcv-and-randomizedsearchcv-e94f53a518ee) article for providing some code used below in the automatic hyperparameter tuning.\n",
+    "Load example data from sklearn"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "id": "91321870",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sklearn import datasets\n",
+    "diabetes_X, diabetes_y = datasets.load_diabetes(return_X_y=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e884e357",
+   "metadata": {},
+   "source": [
+    "Split the data into training, testing, and validation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "id": "be4de1fa",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from astartes import train_val_test_split\n",
+    "X_train, X_val, X_test, y_train, y_val, y_test = train_val_test_split(diabetes_X, diabetes_y)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0c288eb8",
+   "metadata": {},
+   "source": [
+    "Create a baseline model without tuning it for better performance on our data:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "id": "6b73362c",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0.3515039651924021"
+      ]
+     },
+     "execution_count": 26,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from sklearn.ensemble import RandomForestRegressor\n",
+    "rfr_baseline = RandomForestRegressor(n_estimators=5)\n",
+    "rfr_baseline.fit(X_train, y_train)\n",
+    "rfr_baseline.score(X_val, y_val)"
```

Comment:
Could you please add a comment that says what `.score()` returns? Admittedly I forgot what RF's default was and it would've been helpful to know it was the coefficient of determination.

Reply:
Done